packages/react/src/Card/Card.features.stories.tsx

```diff
@@ -31,19 +31,67 @@ export const Label: ComponentStory<typeof Card> = () => {
     </Card>
   )
 }
+
+export const Icon: ComponentStory<typeof Card> = () => {
+  return (
+    <Card href="https://github.com">
+      <Card.Icon icon={SearchIcon} color="purple" hasBackground />
+      <Card.Heading>Code search & code view</Card.Heading>
+      <Card.Description>
+        Enables you to rapidly search, navigate, and understand code, right from GitHub.com.
+      </Card.Description>
+    </Card>
+  )
+}
+
+export const WithIconSVG = () => (
+  <Card href="https://github.com">
+    <Card.Icon
+      icon={
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="20" height="20">
+          <path
+            d="M10.68 11.74a6 6 0 0 1-7.922-8.982 6 6 0 0 1 8.982 7.922l3.04 3.04a.749.749 0 0 1-.326 1.275.749.749 0 0 1-.734-.215ZM11.5 7a4.499 4.499 0 1 0-8.997 0A4.499 4.499 0 0 0 11.5 7Z"
+            fill="currentColor"
+          ></path>
+        </svg>
+      }
+      hasBackground
+      color="purple"
+    />
+    <Card.Heading>With leading visual</Card.Heading>
```

Comment:
Total nit. This one's the odd one out right now.

```suggestion
    <Card.Heading>Code search & code view</Card.Heading>
```